terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/WebDriverOperations.java

```diff
@@ -33,10 +34,15 @@
 
     protected final WebDriver webDriver;
 
+    protected final WebDriverWait webDriverWait;
```

Comment:
- [ ] I think... It seems better to define`webDriverWait` field at not `WebDriverOperations` but `FunctionTestSupport`.
Because `WebDriverOperations` is a class that wraps a particular operation using `WebDriver`, but the `webDriverWait` field is not wrapped properly.
As with `WebDriver`, it looks appropriate to call `WebDriverWait` defined in the superclass from the test class.

Reply:
c9208751b059a508fe92cc1acd9097e2217619f3
Sorry. Yes, it better to define `webDriverWait` field ad `FunctionTestSupport` .
Fixed at above commit.